src/Arcus.Scripting.LogicApps/Scripts/Disable-AzLogicAppsFromConfig.ps1

```diff
@@ -14,27 +14,31 @@ function ExecuteStopType() {
     param
```

Comment:
This file is starting to get complex. We might refactor some inner functionality to separate functions, maybe? Can be looked at later, of course.

Reply:
Would leave it for a later enhancement indeed 👍🏻